common/src/main/java/net/whimxiqal/journey/data/DataManagerImpl.java

```diff
@@ -76,23 +76,21 @@ public void initialize() {
           }
         }
         break;
-//      case MYSQL:
-//        MySqlConnectionController mysqlController = new MySqlConnectionController();
-//        personalWaypointManager = new SqlPersonalWaypointManager(mysqlController);
-//        publicWaypointManager = new SqlPublicWaypointManager(mysqlController);
-//        pathRecordManager = new SqlPathRecordManager(mysqlController);
-//        tunnelDataManager = new SqlTunnelDataManager(mysqlController);
-//        if (setupSchema) {
-//          try (Connection connection = mysqlController.establishConnection()) {
-//            Statement statement = connection.createStatement();
-//            addBatchesToStatement("/data/sql/schema/mysql.sql", statement);
-//            statement.executeBatch();
-//          } catch (SQLException e) {
-//            setupSchema = false;
-//            e.printStackTrace();
-//          }
-//        }
-//        break;
+      case MYSQL:
+        MySqlConnectionController mysqlController = new MySqlConnectionController();
+        personalWaypointManager = new SqlPersonalWaypointManager(mysqlController);
+        publicWaypointManager = new SqlPublicWaypointManager(mysqlController);
+        pathRecordManager = new SqlPathRecordManager(mysqlController);
+        tunnelDataManager = new SqlTunnelDataManager(mysqlController);
+        try (Connection connection = mysqlController.establishConnection()) {
+          Statement statement = connection.createStatement();
+          addBatchesToStatement("/data/sql/schema/mysql.sql", statement);
+          statement.executeBatch();
```

Comment:
This should only happen when the data version is at `V0`, since we only create the schema from scratch if it hasn't already been created

Reply:
Fixed, I think that would of actually caused issues if published as is as well.